src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -45,6 +45,16 @@ internal abstract partial class CommonCompiler
         public abstract Compilation CreateCompilation(TextWriter consoleOutput, TouchedFileLogger touchedFilesLogger, ErrorLogger errorLoggerOpt);
         public abstract void PrintLogo(TextWriter consoleOutput);
         public abstract void PrintHelp(TextWriter consoleOutput);
+
+        /// <summary>
+        /// Print compiler version
+        /// </summary>
+        /// <param name="consoleOutput"></param>
+        public virtual void PrintVersion(TextWriter consoleOutput)
+        {
+            consoleOutput.WriteLine(GetAssemblyVersion());
```

Comment:
Why assembly version rather than file version? And do we want to include the Git SHA, which is now placed in the informational version?